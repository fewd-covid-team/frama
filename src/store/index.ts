import { configureStore } from '@reduxjs/toolkit';
import countryReducer, {
  pickRU, pickUA, pickUSA,
} from './slice';

export default configureStore({
  reducer: {
    country: countryReducer,
  },
});

export { pickUA, pickUSA, pickRU };
