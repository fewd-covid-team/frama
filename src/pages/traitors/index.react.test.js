import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import TraitorsScreen from './index';
import store from '../../store';

test('TraitorsScreen snapshot', () => {
  const component = renderer.create(
    <Provider store={store}>
      <TraitorsScreen />
    </Provider>,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
