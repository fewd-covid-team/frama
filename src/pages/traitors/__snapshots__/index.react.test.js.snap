// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TraitorsScreen snapshot 1`] = `
<div
  className="sc-bdfBQB ZMMEF"
>
  <div>
    <p
      className="sc-gsTEea fveWXe"
    >
      Betray your friend!
    </p>
  </div>
  <table
    className="sc-dlfnuX gWLTil"
  >
    <thead>
      <tr>
        <th
          className="sc-eCstlR cALRct"
        >
          ID
        </th>
        <th
          className="sc-eCstlR cALRct"
        >
          Name
        </th>
        <th
          className="sc-eCstlR cALRct"
        >
          Last name
        </th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <th
          className="sc-jSgvzq gZqCPI"
        >
          1
        </th>
        <th
          className="sc-jSgvzq gZqCPI"
        >
          Vladimir
        </th>
        <th
          className="sc-jSgvzq gZqCPI"
        >
          Putin
        </th>
      </tr>
    </tbody>
  </table>
  <div
    className="sc-gKseQn eBUiiy"
  >
    <form
      action="#"
      onReset={[Function]}
      onSubmit={[Function]}
    >
      <div>
        <input
          className="sc-iBPTik jxFLwP"
          id="name"
          name="name"
          onBlur={[Function]}
          onChange={[Function]}
          placeholder="First Name"
          value="me"
        />
        <input
          className="sc-iBPTik jxFLwP"
          id="lastName"
          name="lastName"
          onBlur={[Function]}
          onChange={[Function]}
          placeholder="Last Name"
          value="123"
        />
      </div>
      <div
        className="sc-gKseQn eBUiiy"
      />
      <div
        className="sc-gKseQn eBUiiy"
      />
      <div
        className="sc-gKseQn eBUiiy"
      >
        <button
          className="sc-fubCzh kHaUnY"
          type="submit"
        >
          Submit
        </button>
      </div>
    </form>
  </div>
</div>
`;
